### DISCUSSION TITLE
Compression Option?

### DISCUSSION THREAD

**Discussion by 1RonRock** [sienori/Tab-Session-Manager#781] [Q&A]
I would like to try the compression option that has been recently added to TSM. Is it "safe" to use compression? I have had issues previously with TSM, mostly after Firefox changed their format. Seemed to take a while to catch up. But currently FF and TSM seem to be working fine, although a bit slower than I think things should be. So I ask is it safe to use compression or is that just adding another level of complexity that could add to potential problems?

**Comment by sienori** [comment]
I believe compression is safe. No problems have been reported so far.

The `Compress all sessions` option can greatly improve performance.
![image](https://user-images.githubusercontent.com/29514424/119085689-261be380-ba3f-11eb-9b36-2c13cb0b70d7.png)

**Comment by rediffusion** [comment]
I have over 50 tabs. My compression result is 😁
Slight change: 0.13 MB → 0.12 MB
